yarn-project/end-to-end/src/integration_l1_publisher.test.ts

```diff
@@ -253,6 +262,8 @@ describe('L1Publisher integration', () => {
     }
     // Path relative to the package.json in the end-to-end folder
     const path = `../../l1-contracts/test/fixtures/${fileName}.json`;
+
+    // NOTE: Members of the struct (and substructs) have to be in ALPHABETICAL order!
```

Comment:
A note here, I don't think the json actually need to be alphabetical, just that it will be loaded alphabetically.